core-services/src/main/java/org/zalando/nakadi/validation/EventValidatorBuilder.java

```diff
@@ -1,27 +1,84 @@
 package org.zalando.nakadi.validation;
 
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.ValidationException;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import org.zalando.nakadi.domain.EventCategory;
 import org.zalando.nakadi.domain.EventType;
-import org.zalando.nakadi.domain.ValidationStrategyConfiguration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
 
 @Component
 public class EventValidatorBuilder {
 
+    private final RFC3339DateTimeValidator dateTimeValidator = new RFC3339DateTimeValidator();
+    private final JsonSchemaEnrichment loader;
+
+    @Autowired
+    public EventValidatorBuilder(
+            final JsonSchemaEnrichment loader) {
+        this.loader = loader;
+    }
+
     public EventTypeValidator build(final EventType eventType) {
-        final EventTypeValidator etv = new EventTypeValidator(eventType);
+        final List<Function<JSONObject, Optional<ValidationError>>> validators = new ArrayList<>();
```

Comment:
I would not prefer to use array list here at all, because after that you create stream to go over 2 elements, but if you do so, please, set initial array size